### DISCUSSION TITLE
Publish latest commit on `main` as docker image version `dev`

### DISCUSSION THREAD

**Discussion by ringods** [goreleaser/goreleaser#2089] [Q&A] [answered]
I have a working Goreleaser setup here when setting a tag:

https://github.com/ringods/pulumi-resource

When pushing a tag, it creates a Github release and publishes the corresponding Docker image to the Github Container registry with versions `latest` and the tag name.

I just tried adding another config file so the latest commit on branch `main` is published only as a Docker image with version`dev` (so not Github Release).

https://github.com/ringods/pulumi-resource/commit/56e8b877ba9f084520dc652e2fcbd552d0d1794f

But when running this, the image is build but not published. What am I missing?

**Comment by caarlos0** [comment, marked as answer]
Images are not published on snapshots... you can add manually a `docker push` in another step of your CI...
